src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp

```diff
@@ -46,6 +46,7 @@ BlockLocalPositionEstimator::BlockLocalPositionEstimator() :
 	_dist_subs(),
 	_sub_lidar(nullptr),
 	_sub_sonar(nullptr),
+	_sub_beacon_position(ORB_ID(beacon_position), 1000 / 40, 0, &getSubscriptions()),
```

Comment:
this can be replaced by a general `landing_target` structure, which can have data from IR beacons, sound beacons, vision markers, etc.